src/constants/url.ts

```diff
@@ -1,4 +1,6 @@
 export const TINY_API_URL = 'https://staging-tinysite-api.realdevsquad.com/v1';
 export const TINY_API_GOOGLE_LOGIN = `${TINY_API_URL}/auth/google/login`;
 export const TINY_API_LOGOUT = `${TINY_API_URL}/auth/logout`;
-export const BASE_SHORT_URL = 'https://staging-tinysite.realdevsquad.com';
+export const TINY_SITE = 'https://staging-tinysite.realdevsquad.com';
```

Comment:
shall we call the variable by `TINY_SITE_STAGING_URL`

Reply:
https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/43